setup.py

```diff
@@ -75,7 +75,7 @@ def run_tests(self):
         'develop': ["cherrypy==3.2.4", "pyOpenSSL"],
         'testing': tests_requires,
         'docs': ['Sphinx', 'sphinx-autobuild', 'alabaster'],
-        'quality': ['pylama', 'isort', 'eradicate', 'mypy', 'black'],
+        'quality': ['pylama', 'isort', 'eradicate', 'mypy', 'black', 'bandit'],
```

Comment:
I am not sure if this is necessary. I consider this as requirements for quality run in tox and it is not (yet) needed there.

Reply:
Not sure either. If you have a better place to put it, I'm fine with it.

Reply:
Maybe do not put it in requirements at all? Otherwise I do not have any other place so if you want it to be somewhere, then go ahead and merge this.